src/java/org/apache/cassandra/index/sai/IndexContext.java

```diff
@@ -160,7 +160,7 @@ public IndexMetadata getIndexMetadata()
     /**
      * @return A set of SSTables which have attached to them invalid index components.
      */
-    public Collection<SSTableContext> onSSTableChanged(Collection<SSTableReader> oldSSTables, Collection<SSTableContext> newSSTables, boolean validate)
+    public Collection<SSTableContext> onSSTableChanged(Collection<SSTableReader> oldSSTables, Collection<SSTableContext> newSSTables, IndexValidation validate)
```

Comment:
Nit: I'd call the parameter `validation`, instead of the current boolean-style `validate`

Reply:
fixed